extra/1-radio-stations.js

```diff
@@ -26,6 +52,12 @@
  */
 // `getStations` goes here
 
+function getStations (){
+  const frequenciesAvailable = getAllFrequencies();
+  const radioFrequencies = frequenciesAvailable.filter(frequencies => isRadioStation(frequencies));
+  return radioFrequencies;
```

Comment:
Would probably return the expression directly

i.e  return frequenciesAvailable.filter(frequencies => isRadioStation(frequencies));